app/models/date_facet.rb

```diff
@@ -41,7 +41,7 @@ def applied_filters
     present_values.map do |type, date|
       preposition = preposition_mappings[type]
       {
-        name: "#{name} #{preposition}",
+        name: "#{short_name} #{preposition}",
```

Comment:
Why does the `short_name` need to be updated here and in the view? I assumed that seeing `short_name` in the "active filters" section on the page was just presentational?  Or is `name` the key and they have to match?

Reply:
Good question – it's because this and the view control separate things.

This one in `#applied_filters` is for the filter tags:
<img width="254" alt="image" src="https://github.com/user-attachments/assets/962eb507-2503-412d-90fb-450bfeec8a3b">

... and the one in the view is for the input field legend:
<img width="287" alt="image" src="https://github.com/user-attachments/assets/49e61d09-eed4-4988-b84c-f8c595a47fef">

Reply:
Thank you. I missed that both are being updated.